<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nebDialog</class>
 <widget class="QDialog" name="nebDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NEB Plotter</string>
  </property>
  <widget class="QPlainTextEdit" name="nebInput">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>201</height>
    </rect>
   </property>
   <property name="plainText">
    <string>-523.2538805
-522.7895516
-523.0864063
-522.9183905
-523.2192523
-523.389196
-523.0598411
-522.9292259
-523.0128113
-522.842924
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plotBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Energies:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="nebOutput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>121</width>
     <height>201</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="sepCombo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>\n</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>,</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>;</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Separator:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="nebLocationInput">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>71</width>
     <height>201</height>
    </rect>
   </property>
   <property name="plainText">
    <string>0
2
4
6
8
10
12
14
16
18
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Locations:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xUnit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>\AA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yUnit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>eV</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X Unit:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Unit:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="arrowChk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Show only Highest Value</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="labelInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>295</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Path 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>295</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
